plugins/trendmicro_visionone/plugin.spec.yaml

```
@@ -1171,6 +1226,58 @@ triggers:
         type: '[]alert'
         required: true
         example: '["<<referenced:bigdata>>"]'
+  poll_oat_list:
+    title: Poll OAT List
+    description: Polls information about Observed Attack Techniques (OATs) events that match the specified criteria in a list
+    input:
+      detected_start_date_time:
+        title: Detected Start Date Time
+        type: string
+        description: The start of the event detection data retrieval time range in ISO 8601 format
+        required: true
+        example: 2021-04-05T08:22:37Z
+      ingested_start_date_time:
+        title: Ingested Start Date Time
+        type: string
+        description: The beginning of the data ingestion time range in ISO 8601 format
+        required: true
+        example: 2021-04-06T08:22:37Z
+      interval:
+        title: Interval
+        type: integer
+        description: Interval (in seconds) in which the polling script should run again
+        required: true
+        default: 1800
+        example: 1800
+      fields:
+        title: Fields
+        type: object
+        description: JSON object of OAT identifiers to query
+        required: true
+        example: '{"uuid":123,"endpointName":"client1"}'
+      query_op:
+        title: Query Operator
+        type: string
+        description: Logical operator to employ in the query. (AND/OR)
+        required: true
+        example: ' or '
+        default: ' or '
+        enum:
+          - ' or '
+          - ' and '
```

more so just a qq, do you need the extra whitespace here?
```suggestion
        example: ' or '
        default: ' or '
        enum:
          - 'or'
          - 'and'
```